packages/ui/src/forum/components/CategoryCard/CategoryCard.tsx

```diff
@@ -14,10 +14,10 @@ import { ForumCategory } from '@/forum/types'
 export interface CategoryCardProps {
   category: ForumCategory
   className?: string
+  archivedStyles?: boolean
```

Comment:
Please add this to the story control.
Also `isArchive` should be removed from the ForumMain story